src/mgr/Mgr.cc

```diff
@@ -529,6 +529,16 @@ void Mgr::handle_mon_map()
       names_exist.insert(monmap.get_name(i));
     }
   });
+  for (const auto& name : names_exist) {
```

Comment:
```
/home/jenkins-build/build/workspace/ceph-pull-requests/src/mgr/Mgr.cc:539:30: error: 'fmt' has not been declared
  539 |     monc->start_mon_command({fmt::format(cmd, name)}, {},
      |                              ^~~
/home/jenkins-build/build/workspace/ceph-pull-requests/src/mgr/Mgr.cc:540:30: error: no matching function for call to 'MonClient::start_mon_command(<brace-enclosed initializer list>, <brace-enclosed initializer list>, ceph::bufferlist*, std::string*, MetadataUpdate*&)'
```

might want to add
```c++
#include <fmt/format.h>
```

right after `#include <Python.h>` at the beginning of this source file.

Reply:
Yeah Kefu thanks, I know. My Mac just block my work day with last Big Sur update... I fix this on this week